kernel/common/binding.rb

```diff
@@ -54,4 +54,12 @@ def eval(expr, filename=nil, lineno=nil)
 
     Kernel.eval(expr, self, filename, lineno)
   end
+
+  def local_variables
+    eval('local_variables')
```

Comment:
`Binding` already has its `VariableScope` - it's in the `@variables` ivar, from `Binding.setup`.